example/settlement/main.go

```diff
@@ -262,7 +262,7 @@ func main() {
 		NextStop:      "",
 	}
 
-	if blk, err := client.Settlement.GetProcessCDRBlock(&cslAddr, cdr2, func(hash types.Hash) (signature types.Signature, err error) {
+	if blk, err := client.Settlement.GetProcessCDRBlock(&cslAddr, []*qlcchain.CDRParam{cdr2}, func(hash types.Hash) (signature types.Signature, err error) {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
265-288 lines are duplicate of `example/settlement/main.go:229-252` (dupl)